firebase/package.json

```diff
@@ -6,6 +6,7 @@
   "private": true,
   "scripts": {
     "postinstall": "run-p install:*",
+    "setup-ci": "npm i && run-p install:*",
```

Comment:
`postinstall` has some problems when running in sudo users, such as in Docker environments or in CI machines. An alternative would be to use `--unsafe-perm` flag, which is dangerous. So I've decided to create this extra script that does the setup for CI environments